### PR TITLE
[T4A5][F10-B3]Lee Wenwei Johnervan

### PR DESCRIPTION
**Pull request by johnervan**

### CODE REVIEW

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	public static final String DELETED = "-";
+	
+	private Person person;
+	private Tag tag;
+	private Boolean isAdded;
+	
+	public Tagging(Boolean isAdded, Person person, Tag tag){
+		this.isAdded = isAdded;
+		this.person = person;
+		this.tag = tag;
```
Comment (chanjunweimy) [inline src/seedu/addressbook/data/tag/Tagging.java:25]: Please use 4 spaces instead of tab. :)

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+			return ADDED;
+		} else {
+			return DELETED;
+		}
+	}
+	
+	@Override
+	public String toString() {
+        return getChange() + " " + person.getName() + " " + tag.toString();
+    }
```
Comment (chanjunweimy) [inline src/seedu/addressbook/data/tag/Tagging.java:39]: Please indent properly :P

**Comment by chanjunweimy** [comment]
@johnervan Some comments added. Please close the PR after reading comments. Good job! :)

**Comment by johnervan** [comment]
Ok will make the changes. Thanks!

**Pull request closed without merging**